nixos/modules/virtualisation/cri-o.nix

```diff
@@ -61,9 +61,9 @@ in
       network_dir = "/etc/cni/net.d/"
 
       [crio.runtime]
-      conmon = "${pkgs.conmon}/bin/conmon"
```

Comment:
@saschagrunert Could you readd the `conmon` path please, it's needed for #86637?

Reply:
Fixed :+1: